src/main/java/org/folio/circulation/domain/notice/NoticeContextUtil.java

```diff
@@ -149,7 +149,7 @@ private static JsonObject createLoanContext(
 
     JsonObject loanContext = new JsonObject();
     loanContext.put("initialBorrowDate", loan.getLoanDate().withZone(timeZone).toString());
-    loanContext.put("numberOfRenewalsTaken", loan.getRenewalCount());
+    loanContext.put("numberOfRenewalsTaken", Integer.toString(loan.getRenewalCount()));
```

Comment:
Is this being conversion done explicitly because all tokens values are strings?

Reply:
Yes, that is correct.